core/src/components/input/test/label-placement/input.e2e.ts

```diff
@@ -93,4 +121,34 @@ test.describe('input: label placement floating', () => {
       `input-focused-placement-floating-no-value-${page.getSnapshotSettings()}.png`
     );
   });
+  test('long label should truncate', async ({ page }) => {
+    await page.setContent(`
+      <ion-input label="Email Email Email Email Email Email Email Email Email Email Email Email" value="example@ionic.io" label-placement="floating"></ion-input>
+    `);
+    const input = page.locator('ion-input');
+    expect(await input.screenshot()).toMatchSnapshot(
+      `input-placement-floating-long-label-${page.getSnapshotSettings()}.png`
```

Comment:
I noticed that only some of these screenshots show the label as truncated; looks like it's a matter of ios vs md? Is the behavior supposed to be inconsistent like that?

Reply:
Whoops I had some styles only applied to iOS mode. I fixed this in https://github.com/ionic-team/ionic-framework/pull/26252/commits/9ee347212134307ff01a5e73de1ad1298ab39d0b and will update screenshots now.